src/components/dialog/dialog.style.js

```diff
@@ -84,14 +88,17 @@ const DialogTitleStyle = styled.div`
   border-bottom: 1px solid #ccd6db;
   ${({ showCloseIcon }) => showCloseIcon && "padding-right: 85px"};
 
+  ${StyledHeaderContent} {
+    align-items: baseline;
+  }
+
   ${StyledHeading} {
     margin-bottom: 20px;
 
     ${StyledHeadingTitle} {
       color: ${({ theme }) => theme.text.color};
       display: block;
       overflow: hidden;
-      white-space: nowrap;
```

Comment:
How come this needed to be removed?

Reply:
It helps with text wrapping

Reply:
Sorry I didn't read the part of the PR about fixing text wrapping!